apps/vaporgui/main.cpp

```diff
@@ -71,9 +71,7 @@ FILE *OpenLog(string path_var) {
 	return(fp);
 }
 
-#ifndef WIN32
-#warning Qt4 uses deprecated OSX calls which pollute the console with warnings
-#endif
+
```

Comment:
Please undelete this warning.

Reply:
Never mind, but please remove the code that this warning warns about.

Reply:
Since this branch is on Qt5, I think we can remove the warning entirely, no?

Reply:
Yes, but this warning warns about code to hide some deprecated warnings from Qt4 so we need to remove the associated code too.